api/nameguard/nameguard.py

```diff
@@ -397,7 +409,10 @@ async def secure_primary_name(self, address: str, network_name: str) -> SecurePr
         else:
             nameguard_result = await self.inspect_name(network_name, domain)
 
-            if nameguard_result.normalization == Normalization.UNNORMALIZED:
+            if nameguard_result is None:
```

Comment:
@lightwalker-eth Here we need a status for `secure_primary_name`. Do we need to add new one? Now we have:
```    NORMALIZED = 'normalized'
    NO_PRIMARY_NAME = 'no_primary_name'
    UNNORMALIZED = 'unnormalized'```

Reply:
@djstrong Yes I suppose we need a new status for this case. Could you please suggest one? Thank you

Reply:
`UNINSPECTABLE`